src/file_operations.rs

```diff
@@ -107,6 +118,8 @@ impl FileOperations {
             Op::MkDir(dir) => try!(fs::create_dir(dir)),
             Op::MkDirAll(dir) => try!(fs::create_dir_all(dir)),
             Op::GitInit(dir) => try!(self.git_init(dir)),
+            Op::Link(src, dst) => try!(os::unix::fs::symlink(src, dst)),
+            Op::Unlink(file) => try!(fs::remove_file(file)),
```

Comment:
The operation name "Unlink" should be renamed to something that relates better to the actual filesystem call. For example Op::remove(file).